pkg/apis/core/types.go

```diff
@@ -6052,6 +6052,9 @@ const (
 	ResourceQuotaScopePriorityClass ResourceQuotaScope = "PriorityClass"
 	// Match all pod objects that have cross-namespace pod (anti)affinity mentioned
 	ResourceQuotaScopeCrossNamespacePodAffinity ResourceQuotaScope = "CrossNamespacePodAffinity"
+
+	// Match all pvc objects that have volume attributes class mentioned.
+	ResourceQuotaScopeVolumeAttributesClass ResourceQuotaScope = "VolumeAttributesClass"
```

Comment:
Do we need to document that, this scope only works for restricting counting and not size. What is strange is - size can still be specified in this scope, but it has no effect:

```
spec:
  hard:
    count/persistentvolumeclaims: "3"
    size/persistentvolumeclaims: 100Gi
  scopeSelector:
    matchExpressions:
    - operator: In
      scopeName: VolumeAttributesClass
      values:
      - silver
status:
  hard:
    count/persistentvolumeclaims: "3"
    size/persistentvolumeclaims: 100Gi
  used:
    count/persistentvolumeclaims: "1"
```

Reply:
@gnufied add a doc https://github.com/kubernetes/website/pull/50082